org.kde.kweather.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "org.kde.kweather",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kweather",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--system-talk-name=org.freedesktop.GeoClue2"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "kweathercore",
+            "config-opts": [ "-DBUILD_TESTING=OFF" ],
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://invent.kde.org/libraries/kweathercore.git", "branch": "release/21.05", "commit": "97dc22c6d1bc32f1e946c420dd0c102f065e46ed" } ]
+        },
+        {
+            "name": "kquickcharts",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [ "-DBUILD_TESTING=OFF" ],
+            "sources": [ { "type": "git", "url": "https://invent.kde.org/frameworks/kquickcharts.git", "tag": "v5.81.0", "commit": "2f868bf91bca9ff778920e81ffaa43c7c53ad5d2" } ]
```

Comment:
To compare with above a tag + commit is fine. However is upstream retag (it's not supposed to happen, but it is possible) it will fail as well. I personally thing it's ok as moving tag shouldn't happen.